__init__.py

```diff
@@ -1,16 +1,13 @@
 import datetime
-import json
-import importlib.util
 import os
-import sys
 import time
 import requests
+
 from os import path
-from pathlib import Path
-from PIL import Image 
+from mycroft_bus_client import Message
+from neon_utils.log_utils import LOG
```

Comment:
can we use the LOG from ovos_utils here?

Reply:
Good catch; was missing that in requirements too. Fixed.